src/modules/account/components/EditAccount/EditAccount.js

```diff
@@ -17,8 +17,8 @@ const EditAccount = ({ history }) => {
       <EditAccountForm onBack={history.goBack} />
       <SetPasswordSuccess
         encryptedPhrase={currentAccount}
-        headerText={t('Edit account name successful')}
-        contentText={t('You can now download encrypted secret recovery phrase to this effect.')}
+        headerText={t('Edit account name')}
+        contentText={t('Successfully edited the account name, now you can download the encrypted secret recovery phrase to this effect.')}
```

Comment:
its a bit too long could we make it shorter?
```suggestion
        contentText={t('Successfully edited, now you can download the encrypted secret recovery phrase to this effect.')}
```